sys/netgraph/ng_pipe.c

```diff
@@ -814,7 +828,7 @@ pipe_dequeue(struct hookinfo *hinfo, struct timeval *now) {
 		/* Randomly discard the frame, according to BER setting */
 		if (hinfo->cfg.ber) {
 			oldrand = rand;
-			rand = random();
+			rand = prng32() & 0x7fffffff;
```

Comment:
This calls for prng32_bounded() as well.

Reply:
Done.